src/main/webapp/scripts/civic.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Extract the government officials from the civic API response.
+ */
+function extractOfficials(civicObj) {
+  let officials = new Object();
+  const levels = ["administrativeArea1", "administrativeArea2", "country", "international", "locality", "regional", "special", "subLocality1", "subLocality2"];
+  levels.forEach((level) => {
+    officials[level] = new Array();
+  });
+  civicObj.offices.forEach((officeObj) => {
+    const level = officeObj.levels[0];
```

Comment:
[Nit] CONSTANT_CASE

Reply:
This isn't a module-local (top-level) constant.